### PR TITLE
Auto assign to project workflow

### PR DESCRIPTION
**Pull request by service-paradis**
### Take 3 (see #4863 and https://github.com/simple-icons/simple-icons/commit/7181ea0b31adce9364fc6e131471be8e17b0781d)

Add new PR to the Organisation Priorisation Project:

- New PRs to Unprioritised column
- PRs labeled as icon outdated to Priority 1

Now using `pull_request_target`. 🤞

### CODE REVIEW

File: .github/workflows/auto-assign-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
+
+jobs:
+  assign-to-project:
+    runs-on: ubuntu-latest
+    name: Assign to Project
+    steps:
+    - name: Assign pull requests to "Unprioritised"
+      uses: srggrs/assign-one-project-github-action@1.2.0
```
Comment (mondeja) [inline .github/workflows/auto-assign-to-project.yml:15]: Could you update the version to [1.2.1](https://github.com/srggrs/assign-one-project-github-action/releases/tag/1.2.1)?

```suggestion
      uses: srggrs/assign-one-project-github-action@1.2.1
```

File: .github/workflows/auto-assign-to-project.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      if: github.event.action == 'opened'
+      with:
+        project: https://github.com/orgs/simple-icons/projects/2
+        column_name: Unprioritised
+
+    - name: Assign `icon outdated` pull requests to "Priority 1"
+      uses: srggrs/assign-one-project-github-action@1.2.0
```
Comment (mondeja) [inline .github/workflows/auto-assign-to-project.yml:22]: Here also :wrench:

**Review by mondeja — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-assign-to-project.yml (hunk 1 above)
Comment (service-paradis) [reply .github/workflows/auto-assign-to-project.yml]: Looks like yesterday was not a good day for me to do anything good 😅 
Sorry for this, should be fixed now!

File: .github/workflows/auto-assign-to-project.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request_target:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
```
Comment (mondeja) [inline .github/workflows/auto-assign-to-project.yml:7]: @ericcornelissen Would you mind to set again the `AUTO_ASSIGN_WORKFLOW_TOKEN` as an organization secret, if is not enabled yet?

Comment (ericcornelissen) [reply .github/workflows/auto-assign-to-project.yml:7]: Done :+1:

**Review by mondeja — APPROVED** [review]
Perfect. I'm going to wait before of my own opening of a new pull request to merge it, so I'll be able to test it properly, but if someones want to merge and test it, feel free to do it :+1:

**Comment by adamrusted** [comment]
> I'm going to wait before of my own opening of a new pull request to merge it, so I'll be able to test it properly, but if someones want to merge and test it, feel free to do it 👍

Will merge this now, and work on a couple PRs to test it.

**Comment by mondeja** [comment]
It seems that the problem has been that the Docker image [has not been updated](https://github.com/srggrs/assign-one-project-github-action/pull/68) by the action owner before the release.

**Comment by service-paradis** [comment]
> It seems that the problem has been that the Docker image [has not been updated](https://github.com/srggrs/assign-one-project-github-action/pull/68) by the action owner before the release.

Do you think this could be tried again @mondeja? 😅

**Comment by mondeja** [comment]
Yes, it should work now! 👌🏼

**Pull request merged**